content/blog/tikv-3.0ga-release-notes.md

```diff
@@ -0,0 +1,160 @@
+3 - READ - Takes(s): 41.2, Count: 948576, OPS: 23002.5, Avg(us): 101087, Min(us): 506, Max(us): 3573196, 95th(us): 217000, 99th(us): 412000
+1 - UPDATE - Takes(s): 36.1, Count: 49777, OPS: 1378.2, Avg(us): 337220, Min(us): 10130, Max(us): 3153989, 95th(us): 679000, 99th(us): 0
+2 - UPDATE - Takes(s): 38.2, Count: 50322, OPS: 1317.1, Avg(us): 361260, Min(us): 9240, Max(us): 2997818, 95th(us): 708000, 99th(us): 0
+3 - UPDATE - Takes(s): 41.2, Count: 50424, OPS: 1224.5, Avg(us): 364278, Min(us): 8801, Max(us): 3719113, 95th(us): 716000, 99th(us): 0
+```
+
+3.0.0:
+
+```
+1 - READ - Takes(s): 24.1, Count: 948742, OPS: 39384.2, Avg(us): 60633, Min(us): 546, Max(us): 1945108, 95th(us): 150000, 99th(us): 351000
+2 - READ - Takes(s): 19.8, Count: 949315, OPS: 47917.6, Avg(us): 46491, Min(us): 365, Max(us): 1820360, 95th(us): 118000, 99th(us): 290000
+3 - READ - Takes(s): 23.5, Count: 948784, OPS: 40378.4, Avg(us): 56232, Min(us): 470, Max(us): 1700652, 95th(us): 146000, 99th(us): 320000
+1 - UPDATE - Takes(s): 24.0, Count: 50256, OPS: 2094.5, Avg(us): 214978, Min(us): 6402, Max(us): 2080028, 95th(us): 488000, 99th(us): 721000
+2 - UPDATE - Takes(s): 19.7, Count: 49685, OPS: 2520.7, Avg(us): 228188, Min(us): 11063, Max(us): 2081087, 95th(us): 592000, 99th(us): 0
+3 - UPDATE - Takes(s): 23.4, Count: 50216, OPS: 2144.4, Avg(us): 234702, Min(us): 17188, Max(us): 1746959, 95th(us): 597000, 99th(us): 0
+```
+
+$$ \frac{39384.2 + 47917.6 + 40378.4}{26205.6 + 24746.9 + 23002.5} = \frac{127680.2}{73955} = 1.73 $$
+
+TiKV 3.0.0 has approximately **1.73x better read performance while under a 95% read / 5% update workload.**
+
+$$ \frac{2094.5 + 2520.7 + 2144.4}{1378.2 + 1317.1 + 1224.5} = \frac{6759.6}{3919.8} = 1.72 $$
+
+TiKV 3.0.0 has approximately **1.72x better update performance while under a 95% read / 5% update workload.**
+
+### Workload C - 100% Read
+
+2.1.14:
+
+```
+1 - READ - Takes(s): 31.3, Count: 999000, OPS: 31967.8, Avg(us): 91216, Min(us): 495, Max(us): 2107986, 95th(us): 213000, 99th(us): 438000
+2 - READ - Takes(s): 30.3, Count: 999000, OPS: 32946.6, Avg(us): 81638, Min(us): 384, Max(us): 1174158, 95th(us): 188000, 99th(us): 391000
+3 - READ - Takes(s): 29.8, Count: 999000, OPS: 33495.9, Avg(us): 87190, Min(us): 491, Max(us): 1715646, 95th(us): 195000, 99th(us): 384000
+
+```
+
+3.0.0:
+
+```
+1 - READ - Takes(s): 24.0, Count: 999000, OPS: 41599.0, Avg(us): 64914, Min(us): 587, Max(us): 7320130, 95th(us): 137000, 99th(us): 466000
+2 - READ - Takes(s): 18.0, Count: 999000, OPS: 55351.7, Avg(us): 51959, Min(us): 454, Max(us): 1070117, 95th(us): 140000, 99th(us): 333000
+3 - READ - Takes(s): 21.0, Count: 999000, OPS: 47659.2, Avg(us): 59935, Min(us): 440, Max(us): 1768322, 95th(us): 157000, 99th(us): 385000
+```
+
+$$ \frac{41599.0 + 55351.7 + 47659.2}{31967.8 + 32946.6 + 33495.9} = \frac{144609.9}{98410.3} = 1.47 $$
+
+TiKV 3.0.0 has approximately **1.47x better read performance while under a pure read workload.**
```

Comment:
IMO, we should present all these numbers as 2sf, I don't believe we can trust YCSB to be precise to 3.